packages/node/src/compartment/module-transforms.js

```diff
@@ -31,6 +31,17 @@ const evadeDirectEvalExpressions = (source) => {
   return source.replace(/\beval(\s*\()/g, DIRECT_EVAL_REPLACE_FN)
 }
 
+/**
+ * Hashbangs seem to offend SES.
+ *
+ * @param {string} source
+ * @returns {string}
+ */
+const decapitateHashbang = (source) => {
+  // careful with the source map
+  return source.replace(/^#!(.+?)(?=\r?\n)/, '// $1')
```

Comment:
- [ ] create an issue for discussion abt adding this to `@endo/evasive-transform`

Reply:
ref: https://github.com/endojs/endo/issues/2736